package.json

```diff
@@ -40,13 +39,46 @@
   "homepage": "https://github.com/denis-sokolov/remove-github-forks",
   "devDependencies": {
     "ava": "^3.12.1",
-    "eslint": "^7.9.0",
-    "prettier": "^1.19.1"
+    "prettier": "^2.1.2",
+    "xo": "^0.33.1"
   },
   "ava": {
     "files": [
       "test/**/*",
       "!test/lib"
     ]
+  },
+  "xo": {
+    "extends": [
+      "plugin:promise/recommended"
+    ],
+    "prettier": true,
+    "rules": {
+      "camelcase": "off",
+      "ava/no-import-test-files": "off",
+      "ava/no-invalid-end": "off",
+      "ava/no-skip-test": "off",
+      "ava/no-statement-after-end": "off",
+      "eqeqeq": "off",
+      "no-useless-call": "off",
+      "no-var": "off",
+      "prefer-rest-params": "off",
+      "prefer-spread": "off",
+      "promise/always-return": "off",
+      "promise/catch-or-return": "off",
+      "promise/no-callback-in-promise": "off",
+      "promise/prefer-await-to-then": "off",
+      "unicorn/explicit-length-check": "off",
+      "unicorn/prevent-abbreviations": "off",
+      "unicorn/filename-case": [
+        "error",
+        {
+          "case": "camelCase"
+        }
+      ]
+    }
+  },
+  "prettier": {
+    "arrowParens": "avoid"
```

Comment:
Let’s remove this, too, what do you think? The more default, the better for me.

Reply:
I generally don’t like it because they’re optional in JavaScript, but sure.

Reply:
Ah, but that’s the beauty of the formatter defaults: it inconveniences us all a little by getting at least one thing wrong. :-)